caps/eth.md

```diff
@@ -0,0 +1,223 @@
+# The eth/63 protocol
+
+'eth' is protocol on the [RLPx](../rlpx.md) transport that facilitates exchange of
+Ethereum blockchain information between peers. The current protocol version is **eth/63**.
+See end of document for a list of changes in past protocol versions.
+
+### Protocol Semantics
+
+Upon an active session, a Status message must be sent. Following the reception of the
+peer's Status message, the Ethereum session is active and any other messages may be sent.
+
+All known transactions should be sent following the Status exchange with one or more
+Transactions messages.
+
+Transactions messages should also be sent periodically as the node has new transactions to
+disseminate. A node should never send a transaction back to the peer that it can determine
+already knows of it (either because it was previously sent or because it was informed from
+this peer originally).
+
+New blocks are propagated using the NewBlock and NewBlockHashes messages. The NewBlock
+message includes the full block and is sent to a small fraction of connected peers
+(usually sqrt(peers)). All other peers are sent a NewBlockHashes message containing just
+the hash of the new block. Those peers may request the full block later if they fail to
+receive it from anyone within reasonable time.
+
+Blocks are typically re-propagated to all connected peers as soon as basic validity of the
+announcement has been established (e.g. after the proof-of-work check).
+
+### Basic Chain Synchronization
+
+Two peers connect and send their Status message. Status includes the Total Difficulty(TD)
```

Comment:
"Connect" in "two peers connect" needs disambiguation/ clarification. I think what you mean here is that the underlying RLPx transport establishes a connection.